app/models/cross_model_slug.rb

```diff
@@ -25,8 +25,6 @@ def exists?(value)
 
       value = value.downcase
 
-      # Reserved check may be redundant, but allows this to be used outside of Validator
-      return true if ReservedWords.all.include?(value)
```

Comment:
I still am not sure if I make the right call here, but it's the quickest thing to do to unblock community team. cc @jaw6 when you return

Reply:
:thinking: Hmmmm... Mostly, I am confused on how this was even an issue? The `unique_across_models` check is `if: :"#{attribute}_changed?"` — were we renaming an existing `Page` to be `code-of-conduct`?

Reply:
It's creating a new `code-of-conduct`.